package.json

```diff
@@ -3,14 +3,20 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@chakra-ui/icons": "^1.0.6",
```

Comment:
We are using yarn not npm, so you have to restore to previous commit and install dependencies again via yarn.
This is applicable to all of package.json changes - we don't want to create package-lock.json
Check this cheatsheet for CLI commands: https://classic.yarnpkg.com/en/docs/migrating-from-npm/